main.py

```diff
@@ -1 +1,35 @@
-rint("hello")
+import mysql.connector
+from mysql.connector import Error
+from fastapi import FastAPI
+from pydantic import BaseModel
+import uvicorn
+
+app = FastAPI()
+
+db_config = {
+    "host": "35.239.158.202",
+    "user": "root",
+    "password": "7$x[;*X-0qpT[J2n",
+    "database": "test",
+}
+
+connection = mysql.connector.connect(**db_config)
+
+
+class StoreItem(BaseModel):
+    naziv: str
+    cena: int
+    kolicina: int
+
+
+@app.get("/store")
+def view_store():
+    cursor = connection.cursor()
+    cursor.execute("SELECT * FROM test.store")
+    results = cursor.fetchall()
+    cursor.close()
+    return results
```

Comment:
I want response to be in this format:
```
[
{ "id":1,
"name": "Jabuka",
....}
]
```
This means that I want you to format the response so that in POSTMAN you get the response as a list of json objects where key is name of the column.